test/cmdlineTests.sh

```diff
@@ -35,6 +35,12 @@ SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-${REPO_ROOT}/build}
 source "${REPO_ROOT}/scripts/common.sh"
 source "${REPO_ROOT}/scripts/common_cmdline.sh"
 
+AUTOUPDATE=""
+if [ "$1" == "--update" ]
+then
+    AUTOUPDATE="true"
+fi
```

Comment:
If parameters other than the first one get skipped, I think the script should make sure there's only one.
```bash
(( $# <= 2 )) || { printError "Too many arguments"; exit 1; }
(( $# <= 1 || $1 == '--update' )) || { printError "Invalid argument: '$1'"; exit 1; }
```

Reply:
Done. My suggestion needed some tweaks though:
- Looks like `$#` does not count the implied `$0` argument.
- Also, the non-numeric part of the second condition required `[[ ]]` because `(( ))` produces a syntax error when the variable is empty.